src/openfermion/circuits/trotter/circuit_validation.py

```diff
@@ -0,0 +1,61 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+'''Tools to assist in circuit/primitive design'''
+
+from typing import List
+
+import numpy
+from scipy import linalg
+import cirq
+import openfermion
+from openfermion.config import EQ_TOLERANCE
+
+
+def validate_trotterized_evolution(circuit: cirq.Circuit,
+                                   op_list: List['openfermion.QubitOperator'],
```

Comment:
change `op_list` to `ops`. Same reason why we call `qubits` instead of `qubit_list`

Reply:
Done